config/vufind/EDS.ini

```
@@ -504,3 +504,18 @@ DetailPageFormat = 'Long'
 ; Default is false.
 ; Note: Customers with custom catalog databases may need to leave this disabled.
 ;ContentProvider = false
+
+[Catalog]
+; This section enables ILS Driver for records from your catalog database in EDS.
+; set EDSHasCatalog to true and ensure your ILS Driver has been configured
```

A little extra clarification might be worthwhile:

```suggestion
; set EDSHasCatalog to true and ensure your ILS Driver has been configured in config.ini's [Catalog] section.
```